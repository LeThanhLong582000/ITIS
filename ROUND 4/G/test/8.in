7
327
212 311 47 114 297 319 238 34 115 57 48 240 291 255 225 83 102 153 144 57 34 244 179 296 139 298 66 170 237 177 129 97 175 292 322 218 8 26 57 127 61 15 266 122 268 296 166 298 313 120 304 292 69 173 48 291 3 76 163 250 77 227 3 180 137 35 284 170 171 112 251 89 275 255 186 144 12 257 251 314 211 105 211 97 68 66 202 207 84 78 259 236 17 241 284 109 177 112 24 243 184 23 210 102 32 65 152 197 248 210 83 117 39 190 274 305 60 205 292 213 15 295 304 318 33 166 66 178 95 307 257 148 24 305 280 81 250 71 271 294 68 1 251 26 206 173 179 243 192 20 52 83 260 119 42 189 308 82 38 136 26 50 84 154 22 212 247 145 284 126 288 264 120 307 191 160 321 193 136 182 241 74 93 174 50 308 203 10 256 1 66 301 19 139 131 255 299 304 87 186 204 309 233 29 249 107 138 108 169 146 116 208 222 20 175 299 300 216 109 173 189 5 25 287 101 51 251 97 174 327 292 159 303 241 234 217 122 75 204 113 47 303 5 256 132 266 140 183 78 2 31 320 324 5 291 240 25 271 161 279 297 297 125 59 166 106 171 326 196 166 113 280 105 322 99 153 225 282 263 16 163 103 43 68 61 31 240 194 91 173 288 174 248 100 180 10 170 91 140 325 173 220 308 247 205 148 48 103 142 73 53 294 78 137 98 27 261 56 204 210 105 277 120 253 22 96 168 
225
123 161 108 174 222 121 201 4 186 201 224 103 53 101 71 210 84 81 208 122 60 82 86 121 85 189 62 55 174 164 56 29 160 73 19 137 203 72 159 216 196 66 10 213 39 19 17 186 136 16 9 150 107 218 194 181 89 187 221 128 64 223 82 52 175 185 9 150 112 140 225 62 197 14 217 152 68 7 150 26 26 130 196 101 14 99 126 65 121 120 31 212 225 148 221 201 56 97 5 98 40 215 178 179 44 104 221 43 122 202 157 29 129 212 54 205 151 45 168 113 190 139 177 41 28 27 87 142 64 96 127 135 137 193 96 212 29 192 204 162 75 66 97 105 129 169 162 125 152 64 172 3 63 114 89 213 223 29 184 134 184 123 5 201 132 103 28 55 215 173 59 220 25 30 70 129 125 105 106 62 81 213 78 217 28 133 132 113 73 157 138 76 166 4 14 96 162 133 81 5 136 169 40 78 85 38 222 159 183 183 166 209 2 5 183 144 6 116 82 13 27 207 67 70 36 
308
147 51 295 34 109 269 203 288 48 154 274 18 298 222 281 112 277 152 166 303 1 295 28 25 55 55 128 72 49 269 155 134 18 22 217 148 68 211 299 59 3 222 213 67 103 100 141 171 308 149 286 219 154 202 118 4 238 303 76 77 292 218 38 74 285 164 170 107 109 213 122 82 247 40 149 242 71 150 96 47 88 289 268 96 306 115 257 92 148 29 119 130 225 62 184 160 180 8 248 308 97 286 60 181 75 164 18 191 177 154 67 114 193 130 261 277 209 109 230 62 308 219 38 108 35 93 269 137 296 29 291 187 59 81 162 124 104 56 176 166 108 48 254 13 59 186 256 13 104 288 40 296 179 123 141 103 118 88 23 81 5 132 19 231 72 201 1 279 183 202 221 247 287 242 50 135 247 306 293 240 209 68 132 268 56 189 24 298 123 207 247 151 55 150 289 75 105 99 115 224 2 150 44 221 52 175 308 281 198 103 307 274 4 202 230 47 57 183 272 126 274 64 54 201 295 31 183 103 213 98 193 149 192 64 73 165 127 140 296 138 141 85 304 52 288 241 275 163 57 113 129 168 271 77 283 181 212 103 129 159 291 237 163 2 227 23 182 296 254 251 175 153 50 128 25 300 171 114 307 9 202 166 6 110 69 248 214 271 137 35 151 124 304 78 52 238 286 250 53 298 267 97 161 129 124 27 60 156 
371
105 238 227 64 201 163 17 288 327 58 65 100 104 185 185 229 54 230 351 83 215 106 144 214 58 157 239 339 283 44 243 146 210 43 199 147 76 114 300 102 43 93 36 363 151 276 48 160 94 123 229 4 172 337 72 292 282 107 72 105 64 235 239 124 243 193 85 319 85 69 223 198 76 169 159 89 124 203 36 7 366 357 157 23 185 255 71 70 12 352 135 256 63 284 197 173 180 155 256 76 301 357 152 185 319 190 362 177 163 246 125 63 369 13 169 78 240 278 256 205 21 343 30 91 341 334 68 87 8 9 229 294 239 325 160 58 348 347 122 360 303 45 35 218 336 41 3 288 12 258 311 341 19 62 225 10 47 163 98 308 138 122 223 78 366 228 151 336 257 293 85 209 144 96 231 249 278 278 155 344 108 174 56 251 150 133 2 166 185 149 308 176 157 47 73 247 19 314 63 156 317 213 137 326 55 50 365 193 72 295 71 144 177 314 319 138 158 221 82 116 165 330 331 311 112 360 321 285 233 178 10 273 249 343 79 106 158 91 146 222 4 244 190 333 32 253 160 254 78 108 293 235 354 230 116 125 21 139 160 47 359 38 319 185 155 6 285 317 219 269 99 364 199 71 89 355 81 54 358 26 131 286 315 12 60 174 286 266 359 330 200 40 267 328 22 329 252 44 79 73 39 149 106 130 258 202 75 116 150 147 257 43 99 262 159 159 86 347 100 350 45 239 366 124 147 21 118 72 8 150 169 293 134 286 115 62 129 290 46 260 133 242 261 348 255 56 11 194 79 343 371 33 64 44 24 180 93 132 208 299 200 305 271 100 254 149 47 73 42 83 22 
373
61 137 20 340 177 62 234 159 325 234 222 317 178 373 54 268 200 68 213 70 343 53 334 209 201 350 186 347 257 334 73 207 346 183 38 315 243 37 137 121 195 256 291 127 234 371 160 142 229 274 333 23 54 170 296 128 309 151 214 51 63 10 359 68 97 300 56 138 105 88 306 324 338 31 250 1 350 128 35 290 101 286 4 234 175 278 321 323 264 191 57 47 56 162 88 66 317 331 75 254 176 1 125 179 266 92 229 232 191 41 324 332 224 78 44 357 177 197 242 245 223 298 350 3 330 70 82 154 119 195 82 95 205 128 211 29 8 367 272 300 267 189 126 157 158 349 161 194 42 251 337 252 174 213 44 59 135 190 67 340 60 181 261 298 302 172 227 139 160 368 312 316 182 311 143 125 112 346 104 24 72 62 310 10 198 104 25 82 219 19 83 84 318 102 269 75 248 81 349 97 50 105 38 22 162 83 61 262 124 7 371 42 182 344 117 297 189 5 318 344 221 202 290 306 342 31 338 196 34 211 162 360 282 245 349 163 119 124 350 334 56 269 71 220 322 257 121 286 166 298 32 208 290 15 114 223 94 304 299 212 153 132 63 104 222 9 364 252 16 153 189 222 339 25 198 307 277 61 324 133 200 136 209 285 373 256 176 89 338 5 309 150 221 51 191 161 237 177 121 366 73 48 251 336 120 322 320 46 166 300 359 209 226 312 41 142 42 326 263 60 36 160 369 93 289 36 337 92 4 358 195 197 203 239 158 68 178 205 70 133 43 261 2 348 112 289 248 349 367 19 150 207 236 327 67 289 284 346 132 246 293 367 109 302 41 81 113 155 45 79 201 279 238 
398
79 378 174 266 58 162 236 125 148 203 107 261 349 304 238 331 163 11 144 160 115 116 247 215 370 250 309 327 268 162 254 226 256 316 216 387 150 235 142 348 227 182 166 207 63 381 293 204 355 336 317 277 180 224 191 174 351 280 37 66 147 265 147 317 134 277 60 92 134 364 126 144 183 13 313 289 6 84 196 289 341 116 65 208 143 244 87 68 161 234 83 226 234 153 179 307 390 307 47 272 89 147 289 49 54 121 64 292 108 150 82 321 348 353 91 383 229 10 371 296 201 183 71 297 14 166 268 197 255 58 377 107 325 141 224 226 226 383 66 95 54 383 196 338 189 318 183 272 79 118 328 206 57 123 165 298 73 63 299 311 216 74 178 267 304 4 108 154 353 67 361 346 305 387 308 329 29 262 234 285 178 43 149 218 392 190 342 276 162 338 140 398 158 387 46 309 166 388 165 217 200 179 144 255 171 365 119 131 164 392 255 342 157 280 289 26 363 334 353 238 318 209 177 83 204 37 348 321 393 45 202 110 214 125 47 69 233 66 221 161 261 379 9 153 290 116 261 259 394 222 208 29 113 64 375 282 395 293 56 130 355 231 183 366 66 393 282 268 303 361 74 85 384 354 255 387 100 150 24 57 139 11 257 33 360 50 254 340 209 205 295 176 100 262 73 372 32 183 341 18 47 350 216 361 23 193 103 379 189 275 71 280 287 151 144 104 277 238 296 63 376 323 359 274 67 140 175 180 67 239 8 38 57 36 164 96 289 34 47 172 74 42 14 320 118 265 288 255 119 2 189 30 360 143 349 205 393 45 337 74 192 371 219 350 3 348 24 289 205 120 163 308 276 46 203 171 225 183 121 347 196 324 366 33 58 52 352 4 184 62 207 117 335 93 94 111 277 91 
287
274 118 24 120 206 34 45 123 209 12 206 13 131 85 93 267 70 182 154 249 14 278 38 200 160 221 142 88 114 12 92 152 57 68 34 22 26 58 234 186 277 117 258 33 201 228 18 236 132 217 40 281 246 148 8 88 172 282 93 131 176 73 75 172 36 123 122 256 252 178 76 61 221 161 128 278 275 251 247 153 123 65 283 248 265 277 120 202 16 169 199 114 167 257 259 172 145 66 173 105 40 38 104 173 141 87 161 108 65 131 241 66 42 178 29 65 214 11 274 209 159 267 281 108 113 15 158 227 71 45 121 182 104 173 56 44 238 281 270 113 114 250 221 162 17 62 124 245 204 212 25 197 210 95 233 86 257 282 30 143 132 137 195 254 112 14 116 147 111 110 36 138 287 185 275 186 121 60 91 43 37 266 191 255 20 115 146 114 4 6 124 136 30 287 147 108 3 220 203 141 30 200 156 82 8 91 90 63 279 49 32 155 52 138 204 103 48 275 283 69 233 93 40 22 115 12 178 256 141 126 259 272 59 114 29 147 25 268 44 133 98 143 54 168 32 114 141 144 55 198 55 254 150 245 239 97 168 177 217 113 144 208 42 261 266 19 109 10 247 286 80 173 220 177 139 93 259 171 257 61 76 45 277 253 149 26 70 
