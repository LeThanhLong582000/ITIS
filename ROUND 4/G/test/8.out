167
116
151
174
189
201
140
